xcode/Subconscious/Shared/Components/Settings/GatewayURLSettingsView.swift

```diff
@@ -8,11 +8,47 @@
 import SwiftUI
 import ObservableStore
 
+struct PendingSyncBadge: View {
+    @State var spin = false
+    
+    var body: some View {
+        Image(systemName: "arrow.triangle.2.circlepath")
+            .rotationEffect(.degrees(spin ? 360 : 0))
+            .animation(Animation.linear
+                .repeatForever(autoreverses: false)
+                .speed(0.4), value: spin)
+            .task{
+                self.spin = true
+            }
+    }
+}
+
+struct ResourceSyncBadge: View {
+    var status: ResourceStatus
+
+    var body: some View {
+        switch status {
+        case .initial:
+            Image(systemName: "arrow.triangle.2.circlepath")
```

Comment:
Tempted to change this icon... It's sort of a warning state so `exclamationmark.triangle` could work.